src/schemas/tiff.schema.json

```diff
@@ -0,0 +1,12 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://github.com/hubmapconsortium/vitessce/#tiff",
+  "title": "Vitessce tiff data",
+  "type": "object",
+  "definitions": {
+    "height": { "type": "number" },
+    "width": { "type": "number" },
+    "tileSize": { "type": "number" },
+    "channels": {"type": "array"  }
+  }
+}
```

Comment:
This is loose on purpose.  I don't know what the API should look like yet since `zarr` is likely an intermediate step.  For now, I think this is fine.

Reply:
File a TODO to tighten it up?